mocha.css

```
@@ -127,35 +193,35 @@ body {
   display: block;
   float: left;
   margin-right: 5px;
-  color: #c00;
+  color: var(--mocha-pass-icon-color);
```

```suggestion
  color: var(--mocha-fail-icon-color);
```
Otherwise, the failed test icon (`✖`) would incorrectly be colored as `#00d6b2` instead of `#cc0000` (light schema) or `#ff4444` (dark schema).